norsk-henvisningsstandard-for-rettsvitenskapelige-tekster.csl

```diff
@@ -0,0 +1,330 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="nb-NO" demote-non-dropping-particle="display-and-sort" page-range-format="expanded">
+  <info>
+    <title>Norsk henvisningsstandard for rettsvitenskapelige tekster (Norsk - Bokmål)</title>
+    <title-short>Norsk rettsvitenskap</title-short>
+    <id>http://www.zotero.org/styles/norsk-henvisningsstandard-for-rettsvitenskapelige-tekster</id>
+    <link href="http://www.zotero.org/styles/norsk-henvisningsstandard-for-rettsvitenskapelige-tekster" rel="self"/>
+    <link href="http://www.zotero.org/styles/universitetet-i-oslo-rettsvitenskap" rel="template"/>
+    <link href="https://www.uib.no/sites/w3.uib.no/files/attachments/veiledning_for_henvisning_i_juridiske_tekster.pdf" rel="documentation"/>
+    <author>
+      <name>Stian Øby Johansen</name>
+      <email>s.o.johansen@jus.uio.no</email>
+    </author>
+    <contributor>
+      <name>Hans Gunnar Slokvik Lian</name>
+      <email>h.g.s.lian@ub.uio.no</email>
+    </contributor>
+    <category citation-format="note"/>
+    <category field="law"/>
+    <summary>Norwegian standard citation style for citing Norwegian and international legal sources. Based on "Formelle krav til rettsvitenskapelige tekster" (August 2017).</summary>
```

Comment:
@StianOby, the current documentation link links to the "veiledning for henvisning i juridiske tekster" from February 2019 (https://www.uib.no/sites/w3.uib.no/files/attachments/veiledning_for_henvisning_i_juridiske_tekster.pdf), not the "Formelle krav til rettsvitenskapelige tekster" from August 2017 (https://www.uio.no/for-ansatte/enhetssider/jus/ledelses-og-utvalgsmoter/pmr/moter/2018/pmr5/vedlegg/sak9.pdf). Can you please clarify which style title and documentation link is correct?

Reply:
I have updated the summary to clarify. Sorry for the confusion.

The standard was slightly updated after a hearing, and the revised version was published in February 2019. I did not catch that they actually changed the title of the style guide at the same time (although the retain the name "Norsk henvisningsstasndard for rettsvitenskapelige tekster" for the citation/bibliography style strictu sensu).